requests/behave.yml

```diff
@@ -0,0 +1,7 @@
+action: token_reset
+feedstocks:
+- behave
+# can be a list with azure, travis, circle, drone, appveyor, github_actions
+skip_providers: []
+# we can expire tokens earlier if desired - default is null which is immediately
+# existing_tokens_time_to_expiration: null  # set to an int in seconds to use
```

Comment:
Are these lines needed for a token reset for all? My guess is that you only need it if you are doing  a specific service, no?